README.md

```diff
@@ -120,6 +122,84 @@ SolidTelemetry.configure do |config|
 end
 ```
 
+## Metrics
+
+By default, SolidTelemetry shows 5 charts:
+
+* CPU usage
+* Memory usage
+* Response time
+* Throughput
+* ActiveJob throughput
+
+You can change the order, remove metrics or add custom metrics in the `config/initializers/solid_telemetry.rb` file.
+
+### Custom metrics
+
+In SolidTelemetry there are 2 kinds of metrics: metrics that measure something with [instruments](https://github.com/open-telemetry/opentelemetry-ruby/tree/main/metrics_sdk/lib/opentelemetry/sdk/metrics/instrument) (cpu load, memory usage, etc), and metrics that measure data from somewhere else, tipically traces (but could be your own app's data).
+
+In both cases, you need to subclass the `SolidTelemetry::Metrics::Base` class and define a few things:
+
+```ruby
+class RandomMetric < SolidTelemetry::Metrics::Base
+  name "random" # self-explanatory
+  description "A random metric for demonstration purposes" # self-explanatory
+  unit "things" # Optional. Used by OpenTelemetry internally, doesn't have a specific use.
+  instrument_kind :gauge # Optional. What kind of instrument are we using to measure. Don't define it if you are looking data elsewhere.
+
+  # Optional. Only define it if you set instrument kind. This means that metrics will be measured periodically.
+  # If defined, SolidTelemetry will store this value as a SolidTelemetry::Metric.
+  def measure
+    rand 100
+  end
+
+  # Optional. Only define it if you don't set an instrument kind and want to measure data elsewhere.
+  def self.metric_data(host, time_range, resolution)
+    # ...
+  end
+end
+```
+
+Then, you need to add this class to the `config/initializers/solid_telemetry.rb` file:
+
+```ruby
+SolidTelemetry.configure do |config|
+  # ...
+  config.metrics = {
+    cpu: [SolidTelemetry::Metrics::Cpu],
+    # ...
+    random: [RandomMetric]
+  }
+end
+```
+
+The `metrics` setting must be a hash, where the key is the chart name and the value is an array of metrics that will be shown in that chart.
+
+#### Example
+
+Let's say you want to track how many orders are created in your e-commerce app. You can do it in two different ways. You can store the measured value in the database or lookup the information every time the metrics are displayed. **Be careful**, this could slow down the metrics page significantly.
```

Comment:
One of the two ways of measuring could slow down the page or either? Not clear to me